src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { TextField as TextFieldMUI, TextFieldProps } from '@mui/material';
+import { InputAdornment } from '@mui/material';
+
+export const TextField = ({
+  sx,
+  placeholder,
+  id = 'outlined-basic',
+  label = 'Headline',
+  variant = 'outlined',
+  disabled = false,
+  color = 'primary',
+  ...props
+}: TextFieldProps) => {
+  return (
+    <TextFieldMUI
+      fullWidth
+      sx={sx}
+      label={label}
+      variant={variant}
+      id={id}
+      disabled={disabled}
+      color={color}
+      placeholder={placeholder}
+      InputProps={{
+        startAdornment: (
+          <InputAdornment position="start">
+            <i className="ph-magnifying-glass"></i>
+          </InputAdornment>
```

Comment:
The default text field shouldn't have an icon since we will use it in different contexts. You can remove this, there is still the withIcon story to demonstrate the use of icons.

Reply:
Agreed. Removed, Done